client/src/plugins/version-info/VersionInfoOverlay.js

```diff
@@ -14,7 +14,7 @@ import { Overlay, Section } from '../../shared/ui';
 
 import { ReleaseInfo } from './ReleaseInfo';
 
-const RELEASE_NOTES_LINK = 'https://camunda.com/blog/category/modeling/?tag=camunda-modeler&release-note&utm_source=modeler&utm_medium=referral';
+const RELEASE_NOTES_LINK = 'https://camunda.com/blog/category/releases/?s=desktop+modeler';
```

Comment:
Seems like the old link redirects to `https://camunda.com/blog/category/releases/?tag=camunda-modeler` instead of `camunda-modeler`. Is this difference relevant for us?

Reply:
Well, technically there is Camunda Web Modeler and Camunda Desktop Modeler. There isn't really blog posts on web modeler so it works out but it's just safer to use desktop modeler

Reply:
:+1: